draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -917,6 +917,13 @@ Example use cases include:
    all packets can be scheduled  over all paths leading to high capacity and
    and high energy efficiency.
 
+The default behavior is, that a path can always be used for packet scheduling 
```

Comment:
I think we should rename MP_PRIO=3 to primary and remove the "always" in the sentence above.